src/test/java/systemtests/FindCommandSystemTest.java

```diff
@@ -1,176 +1,192 @@
+        assertCommandSuccess(command, expectedModel);
+        assertSelectedCardUnchanged();
+
+        /* Case: find person in Anakin, keyword is substring of name -> 0 persons found */
+        command = FindCommand.COMMAND_WORD + " Joh";
+        ModelHelper.setFilteredDeckList(expectedModel);
+        assertCommandSuccess(command, expectedModel);
+        assertSelectedCardUnchanged();
+
+        /* Case: find person in Anakin, name is substring of keyword -> 0 persons found */
+        command = FindCommand.COMMAND_WORD + " Johnny";
+        ModelHelper.setFilteredDeckList(expectedModel);
+        assertCommandSuccess(command, expectedModel);
+        assertSelectedCardUnchanged();
+
+        /* Case: find person not in Anakin -> 0 persons found */
+        command = FindCommand.COMMAND_WORD + " Julius";
+        assertCommandSuccess(command, expectedModel);
+        assertSelectedCardUnchanged();
+        // TODO: Write Card level checks for find
+        //        /* Case: find phone number of person in Anakin -> 0 persons found */
+        //        command = FindCommand.COMMAND_WORD + " " + DECK_D.getPhone().value;
+        //        assertCommandSuccess(command, expectedModel);
+        //        assertSelectedCardUnchanged();
+        //
+        //        /* Case: find address of person in Anakin -> 0 persons found */
+        //        command = FindCommand.COMMAND_WORD + " " + DECK_D.getAddress().value;
+        //        assertCommandSuccess(command, expectedModel);
+        //        assertSelectedCardUnchanged();
+        //
+        //        /* Case: find email of person in Anakin -> 0 persons found */
+        //        command = FindCommand.COMMAND_WORD + " " + DECK_D.getEmail().value;
+        //        assertCommandSuccess(command, expectedModel);
+        //        assertSelectedCardUnchanged();
+        //
+        //        /* Case: find tags of person in Anakin -> 0 persons found */
+        //        List<Tag> tags = new ArrayList<>(DANIEL.getTags());
+        //        command = FindCommand.COMMAND_WORD + " " + tags.get(0).tagName;
+        //        assertCommandSuccess(command, expectedModel);
+        //        assertSelectedCardUnchanged();
+
+
+        /* Case: find person in empty Anakin -> 0 persons found */
+        deleteAllDecks();
+        command = FindCommand.COMMAND_WORD + " " + KEYWORD_MATCHING_JOHN;
+        expectedModel = getModel();
+        ModelHelper.setFilteredDeckList(expectedModel, DECK_D);
+        assertCommandSuccess(command, expectedModel);
+        assertSelectedCardUnchanged();
+
+        /* Case: mixed case command word -> rejected */
+        command = "FiNd Meier";
+        assertCommandFailure(command, MESSAGE_UNKNOWN_COMMAND);
+    }
+
+    /**
+     * Executes {@code command} and verifies that the command box displays an empty string, the result display
+     * box displays {@code AddressbookMessages#MESSAGE_PERSONS_LISTED_OVERVIEW} with the number of people in the
```

Comment:
Seems like it still linked to AB's classes
```suggestion
     * box displays {@code Messages#MESSAGE_DECKS_LISTED_OVERVIEW} with the number of deck in the
```